tests/unit/test__dataframe.py

```diff
@@ -52,26 +53,26 @@
 
 
 def parse_timestamps():
-    from google.api_core import datetime_helpers
-
-    return [datetime_helpers.from_rfc3339(t).replace(tzinfo=None) for t in TIMESTAMPS]
+    return [pandas.Timestamp(t) for t in TIMESTAMPS]
```

Comment:
pandas seems to have its own timestamp type (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Timestamp.html), so I'm not sure how these tests used to work 🤔